### ISSUE TITLE
Option to skip if output file already exists

### ISSUE THREAD

**Issue by zachleat** [11ty/eleventy-img#36]
Must be off by default, but would work nicely with new watch arguments that tell what files changed.

Maybe also check that the image width is the same for an extra level? Not sure here yet how deep to go, would be very confusing to change the source file and the output doesn’t change.

**Comment by reubenlillie** [comment]
How about watching for changes to the arguments to `Image()`? It’d be great if output could be skipped for changes to the `return` statement at least.

**Comment by awesomephant** [comment]
I found that you can pass ```limitInputPixels: 1``` to sharp in order to skip the actual image processing but still return the correct metadata as a workaround. It throws an error to the console, but seems to work.

I'm using an environment variable to do this during development:

```js
eleventyConfig.addShortcode("img", async function (src) {
  let config = {
    widths: [500, 1000],
    formats: ["webp"],
  };
  if (process.env.NODE_ENV === "dev") {
    config.sharpOptions = { limitInputPixels: 1 };
  }
  let metadata = await Image(src, config);
})
```

**Comment by zachleat** [comment]
Shipped with 0.7.0

Docs here: https://www.11ty.dev/docs/plugins/image/#in-memory-cache-new-in-image-0.7.0
